components/Result.jsx

```diff
@@ -1,15 +1,98 @@
 import React from "react";
-import Head from "next/head";
-import { Header } from "./Header";
+import { Bar } from "react-chartjs-2";
+import styled from "styled-components";
 
-export default function Result() {
+export default function Result({ inputs }) {
   return (
-    <div>
-      <Head>
-        <title>Results</title>
-      </Head>
-
-      <Header pageName={"Results"} />
-    </div>
+    <>
+      <StyledDiv>
+        <Bar
+          data={{
+            labels: inputs.map((input) => {
+              return input.value;
+            }),
```

Comment:
you could use shorthand syntax for implicit return + object destructuring, but this is totally fine as is :)

```suggestion
            labels: inputs.map(({ value }) => value),
```